code/datums/ai_law_sets.dm

```diff
@@ -25,7 +25,7 @@
 
 /******************** Nanotrasen/Malf ********************/
 /datum/ai_laws/nanotrasen
-	name = "NT Default"
+	name = "NT Recommended"
```

Comment:
Doesn't excuse the change though.

(Also I made that exactly same argument on the PR removing Nt Default from being a Default module, so preaching to the converted there, my friend)